elastic/apm-server
# Stop overriding logging.{ecs,json} defaults

## Motivation/summary

Stop overriding the defaults for `logging.ecs` and `logging.json`. These are now both forced on by libbeat (https://github.com/elastic/beats/pull/28573).

## Checklist

~- [ ] Documentation has been updated~ (Relevant docs are copied from beats.)

## How to test these changes

1. Run `apm-server -e`
2. Check logs are ECS+JSON

## Related issues

https://github.com/elastic/beats/pull/28573
